### DISCUSSION TITLE
Chakra UI stable enough to build a small production-level application on with React 19 (RC) and Next.js 15?

### DISCUSSION THREAD

**Discussion by AdminForIinRange** [chakra-ui/chakra-ui#9276] [Q&A] [answered]
Hi community,

I’m evaluating Chakra UI for a new project and wanted to know if it’s stable enough to build a small production-level application on, specifically with React 19 (RC) and Next.js 15, I’ve run into a few minor errors, but nothing too crazy, most of them are manageable or ignorable. 

Are there any CRAZY concerns or UNFIXABLE limitations I should be aware of before committing to it for a company-scale product?

Looking forward to your insights!

**Comment by segunadebayo** [comment, marked as answer]
There are no major concerns on our side. Give it a shot and let us know if you run into any issues.
